src/main/java/com/sprint/mission/discodeit/service/jcf/JCFChannelRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.sprint.mission.discodeit.service.jcf;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+
+import java.util.*;
+
+public class JCFChannelRepository implements ChannelRepository {
+    private final Map<UUID, Channel> data = new HashMap<>();
+
+    @Override
+    public Channel create(Channel channel) {
+        data.put(channel.getChannelId(), channel);
+        return channel;
+    }
+
+    @Override
+    public Channel read(UUID id) {
+        return data.get(id);
+    }
+
+    @Override
+    public List<Channel> readAll() {
+        return new ArrayList<>(data.values());
+    }
+
+    @Override
+    public Channel update(UUID id, Channel channel) {
+        if (data.containsKey(id)) {
+            data.put(id, channel);
+            return channel;
+        }
+        return null;
```

Comment:
[Map.computeIfPresent](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfPresent-K-java.util.function.BiFunction-)를 사용해보는건 어떨까요?
```suggestion
        return data.computeIfPresent(id, (key, oldValue) -> updatedUser);
```